RecFCCeeCalorimeter/tests/options/ALLEGRO_o1_v03.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# set-up the Key4hep environment if not already set
+if [[ -z "${KEY4HEP_STACK}" ]]; then
+  source /cvmfs/sw-nightlies.hsf.org/key4hep/setup.sh
+else
+  echo "The Key4hep stack was already loaded in this environment."
+fi
+
```

Comment:
This should check if there is Key4hep stack sourced and fail if it is not. Sourcing of the stack is handled externally.

Reply:
I copied this piece of code from this test https://github.com/HEP-FCC/FCC-config/blob/c5019faefe2768e5ce89ce03c7a581ca5701e892/FCCee/FullSim/ALLEGRO/ALLEGRO_o1_v03/ctest_sim_digi_reco.sh#L3
I guess that @BrieucF had some good reasons for this?